Artifacts/linux-install-mongodb/linux_install_mongodb_v1.sh

```diff
@@ -1,10 +1,14 @@
 #!/bin/bash
 
-set -e
+echo "Installing MongoDb"
 
 isApt=`command -v apt-get`
 isYum=`command -v yum`
 
+# The above commands may error on some flavors depending on which installation mechanism is available.
```

Comment:
I'd say: Some of the previous commands will fail with an exit code other than zero (intentionally), so we do not set error handling to stop (set e) until after they've run.